werobot/messages.py

```diff
@@ -1,63 +1,124 @@
 # -*- coding: utf-8 -*-
 
-MESSAGE_TYPES = {}
-
+import six
 
-def handle_for_type(type):
-    def register(f):
-        MESSAGE_TYPES[type] = f
-        return f
-    return register
+MESSAGE_TYPES = {}
 
 
+class MessageMetaClass(type):
+    def __new__(mcs, name, bases, attrs):
+        for k, v in attrs.items():
+            if isinstance(v, MessageEntry):
+                pass
```

Comment:
Sorry for my weak codes:(
I haven't finished all functions and I know these code is not graceful.I create this pull request only because @whtsky wants to have a look.
Whtsky had a discuss with me and we decided to put the unit test on the top of task list.So I'll close this pr and reopen it after further coding.
@yetone